src/os/ui/providerimport.js

```diff
@@ -38,9 +38,30 @@ os.ui.ProviderImportCtrl = function($scope, $element) {
    */
   this.dp = this.scope['config']['provider'] || null;
 
+  /**
+   * HTML ID for the Format Help windows
+   * @type {string}
+   */
+  this.helpWindowId = 'url-help';
+
+  /**
+   * The help UI for this provider.
+   * @type {string}
+   */
+  this['helpUi'] = null;
+
   this.initialize();
 
   $scope.$emit(os.ui.WindowEventType.READY);
```

Comment:
Our change to reuse the form directive caused some unfortunate window positioning behavior when loading a URL from the Import Data dialog. Because the new form directive is loaded asynchronously with `templateUrl`, the window content isn't fully populated when the `READY` event is fired which means the window height isn't correct. The end result is the window is no longer centered. You can reproduce by entering this URL in the Import Data dialog:

https://demo.geo-solutions.it/geoserver/ows

To fix this, please fire the `READY` event inside a `setTimeout` callback. That will allow the rest of the template to load before the event is fired.